zephyr/CMakeLists.txt

```diff
@@ -894,8 +894,11 @@ set(VERSION_H_PATH ${GENERATED_DIRECTORY}/sof_versions.h)
 find_package(Python3 COMPONENTS Interpreter)
 set(PYTHON3 "${Python3_EXECUTABLE}")
 
-# SOF uses GNU C99 extensions. TODO other flags required ?
+# We don't know why we have this https://github.com/thesofproject/sof/issues/5212
```

Comment:
I think this was a limitation around earlier version of XCC that we no longer use. Pls check the binaries for TGL on Zephyr build, may not be needed anymore.

Reply:
@lgirdwood I already did some analysis in https://github.com/thesofproject/sof/issues/5212 and it was not a quick thing. This PR is merely adding the link so earlier work does not get lost.